lib/jekyll-contentful/client.rb

```diff
@@ -14,7 +14,11 @@ def initialize(args: [], site: nil, options: {})
 
       def sync!
         nfo = "#{client.configuration.dig(:space)} (#{client.configuration.dig(:environment)})"
-        log("Syncing content from Contentful API... #{nfo}\n", color: "green")
+
+        str = "Syncing content from Contentful API"
+        str += " for distribution channels: #{distribution_channels}" if distribution_channels
```

Comment:
There is no universe were putting the decision logic behind the operator will make sense to me.  Not asking for a change. Just complaining to complain.